packages/lambda-functions/project.json

```diff
@@ -4,17 +4,43 @@
   "sourceRoot": "packages/lambda-functions",
   "projectType": "application",
   "targets": {
-    "serve-functions": {
-      "command": "sam.cmd build && sam.cmd local start-api",
+    "build": {
+      "executor": "@nx/webpack:webpack",
+      "outputs": ["{options.outputPath}"],
       "options": {
-        "cwd": "packages/lambda-functions"
+        "compiler": "babel",
+        "target": "node",
+        "externalDependencies": "none",
+        "outputPath": "dist/packages/lambda-functions",
+        "main": "packages/lambda-functions/functions/hello-world/app.ts",
+        "tsConfig": "packages/lambda-functions/tsconfig.json",
+        "assets": [
+          {
+            "input": "packages/db/src/",
+            "glob": "schema.prisma",
+            "output": "/"
```

Comment:
I tried updating this to `/var/task/` to see if that would resolve the error I was seeing, but when I check the docker files there is only `app.js` in `/var/task`

Reply:
`/var/task` is a directory within the docker container where the source code from outside the container is mounted. That's why you aren't seeing anything different when you do this. If you have a /var/task directory on your computer, that's where this went, but the docker container can't see that.